crates/bdk/src/wallet/mod.rs

```diff
@@ -83,19 +86,18 @@ const COINBASE_MATURITY: u32 = 100;
 pub struct Wallet<D = ()> {
     signers: Arc<SignersContainer>,
     change_signers: Arc<SignersContainer>,
-    keychain_tracker: KeychainTracker<KeychainKind, ConfirmationTime>,
-    persist: persist::Persist<KeychainKind, ConfirmationTime, D>,
+    chain: LocalChain,
+    indexed_graph: IndexedTxGraph<ConfirmationTimeAnchor, KeychainTxOutIndex<KeychainKind>>,
+    persist: Persist<D, ChangeSet>, // [TODO] Use a different `ChangeSet`
```

Comment:
What does this TODO mean? Can you remove any TODOs you've made during this PR and make an issue for it and reference the issue instead of just `//TODO` in code.

Reply:
This TODO is not needed, there is nothing to TODO here. Removed.